### PR TITLE
`LSTMDetection` metric crashes when there are multiple context columns

### LINKED ISSUE #298: `LSTMDetection` metric crashes when there are multiple context columns
### Environment Details

Please indicate the following details about the environment in which you found the bug:

* SDMetrics version:
* Python version:
* Operating System:

### Error Description
metadata1={'fields': {'Date': {'type': 'datetime'},
  'Symbol': {'type': 'id'},
  'Open': {'type': 'numerical'},
  'Close': {'type': 'numerical'},
  'Volume': {'type': 'numerical'},
  'Sector': {'type': 'categorical'},
  'Industry': {'type': 'categorical'},
  'MarketCap': {'type': 'numerical'}},
 'entity_columns': 'Symbol',
 'sequence_index': 'Date',
 'context_columns': [
                "MarketCap",
                "Sector",
                "Industry"
            ]}
I am using the same example that was mentioned in Time series data generation using PAR models https://sdv.dev/SDV/user_guides/timeseries/par.html.... I am am unable to evaluate the synthetic data generated using LSTM detection.
lstmerror (1)

### Steps to reproduce

```
Paste the command(s) you ran and the output.
If there was a crash, please include the traceback here.
```

### PR DESCRIPTION
**Pull request by frances-h**
Resolve #298 
CU-860pnm1b0

### CODE REVIEW

**Comment by sdv-team** [comment]
Task linked: [CU-860pnm1b0 SDMetrics - error while using LSTMDetection #298](https://app.clickup.com/t/860pnm1b0)

**Comment by rwedge** [comment]
Can we add a test case to highlight the bugged scenario?

**Review by rwedge — APPROVED** [review]
@frances-h LGTM, assuming my most recent changes look good to you

File: tests/integration/timeseries/test_timeseries.py [hunk 1]
```diff
@@ -44,3 +44,26 @@ def test_compute_all():
     scores = output[output.normalized_score.notna()]
 
     assert scores.normalized_score.between(0.0, 1.0).all()
+
+
+def test_compute_lstmdetection_multiple_categorical_columns():
+    """"""
```
Comment (pvk-developer) [inline tests/integration/timeseries/test_timeseries.py:50]: Missing docstring

**Pull request merged**